packages/ui/src/styles/typography/_mixins.scss

```diff
@@ -183,5 +185,9 @@ $va-display-margins: (
       color: var(--va-link-color);
       filter: brightness(125%);
     }
+
+    &:focus-visible {
+      //@include focus-outline;
+    }
```

Comment:
Is it on purpose?

Reply:
Yeah, my bad - disabled it while testing landing & docs. Thanks.